src/ApplicationRunner.php

```diff
@@ -22,75 +22,27 @@
  */
 abstract class ApplicationRunner implements RunnerInterface
 {
-    protected ?ConfigInterface $config = null;
-    protected ?ContainerInterface $container = null;
-    protected ?string $bootstrapGroup = null;
-    protected ?string $eventsGroup = null;
+    private ?ConfigInterface $config = null;
+    private ?ContainerInterface $container = null;
 
     /**
      * @param string $rootPath The absolute path to the project root.
      * @param bool $debug Whether the debug mode is enabled.
-     * @param string $paramsConfigGroup The config parameters group name.
-     * @param string $containerConfigGroup The container configuration group name.
+     * @param string|null $configGroupPostfix A configuration groups postfix.
      * @param string|null $environment The environment name.
      */
     public function __construct(
         protected string $rootPath,
         protected bool $debug,
-        protected string $paramsConfigGroup,
-        protected string $containerConfigGroup,
-        protected ?string $environment
+        protected bool $useBootstrap,
+        protected bool $checkEvents,
+        protected ?string $configGroupPostfix,
```

Comment:
What's the usage for it?

Reply:
Base names of configs is `di`, `di-providers`, `bootstrap`, `params` and etc. In application postfix passed as parameter to runner. In most cases it's will be `web` or `console`.

Runner for each of used  configs try add postfix, and if exists then use config with postfix and without postfix otherwide.

For example for `di` with postfix `web`.

- Case 1. In app `di` exists and `di-web` exists: use `di-web`.
- Case 2. In app `di` exists and `di-web` not exists: use `di`.
- Case 3. In app `di` not exists and `di-web` not exists: runner do not set container definitions.

Reply:
Unfortunately, `Config` not available check existing of params group (for check need instance of `Config`, but params group must be passed to constructor). I made an explicit parameters of all group names in `Application` constructor.